### ISSUE TITLE
where to put the project video file ?

### ISSUE THREAD

**Issue by hanselhermawan** [uglymie/lane-line-detection-ROS2-CPP#1]
I put the "/data/road_video/project_video.mp4" inside the 'cpp_advanced_lane_lines' file and I did already change the capture format with the correct path. But why is the video does not open in the image view in rqt ? 

Also the in the terminal there is a warning about pipeline something like this :
![image](https://github.com/uglymie/lane-line-detection-ROS2-CPP/assets/117617089/7f120ed3-325c-4c8b-869e-24c9da68202d)

**Comment by uglymie** [comment]
The absolute file path of your video is displayed as '/oooooo/src/...'. Are you sure it is located at this path?

**Comment by hanselhermawan** [comment]
yes, the path is already correct. should i open the video mp4 first or just launch the code and the video mp4 will open up automatically ?

![image](https://github.com/uglymie/lane-line-detection-ROS2-CPP/assets/117617089/25fd6edb-5a38-4be2-ac9c-7c195a87c3e6)

**Comment by uglymie** [comment]
After configuring the path of the video file, you can directly start the program. However, the output below shows that your video has a frame count of 0, which means that your video has not been opened or there is an error in the video file. Can you double check your video file to see if it can be opened with a video player
